keyless/pepper/common/src/lib.rs

```diff
@@ -111,6 +123,11 @@ pub struct PepperResponseV1 {
         deserialize_with = "deserialize_bytes_from_hex"
     )]
     pub signature_encrypted: Vec<u8>, // unique BLS signature encrypted
+    #[serde(
+        serialize_with = "serialize_bytes_to_hex",
+        deserialize_with = "deserialize_bytes_from_hex"
+    )]
+    pub pepper_encrypted: Vec<u8>, // unique BLS signature encrypted
```

Comment:
I thought we are removing this.

Reply:
Yes we will, i just didn't do it in this PR

Reply:
Oh right nvm. removed